reference/shipping.v3.yml

```diff
@@ -410,6 +410,16 @@ components:
           example: true
         hs_codes:
           $ref: '#/components/schemas/harmonizedSystemCodes'
+        createdAt:
+          type: string
+          description: Date and time of the customs information's creation. Read-Only.
```

Comment:
Is there a better approach to marking a field as read-only than just updating the description?

For example, I see in Tax Properties documentation we have `readOnly: true`. https://github.com/bigcommerce/api-specs/blob/9e603bdb4ca15b38b343acf0f86d719332e37f76/reference/tax_properties.v3.yml#L297

Would be worth updating the other existing item in this yml file too with this better approach, while we are here.

Reply:
Yes, the readOnly flag looks better, thanks for mentioning. Just added them to all the read-only fields.